nodejs/eks/examples/oidc-iam-sa/index.ts

```diff
@@ -109,25 +110,24 @@ const bucket = new aws.s3.Bucket("pod-irsa-job-bucket", {
 const bucketName = bucket.id;
 const regionName = pulumi.output(aws.getRegion({}, {async: true})).name;
 const podName = `${saName}-pod-test`;
-const s3Pod = pulumi.all([bucketName, regionName]).apply(([bName, region]) => {
-    return new k8s.core.v1.Pod(podName,
-        {
-            metadata: {labels: labels, namespace: appsNamespaceName},
-            spec: {
-                serviceAccountName: sa.metadata.name,
-                containers: [
-                    {
-                        name: podName,
-                        image: "amazonlinux:2018.03",
-                        command: ["sh", "-c",
-                            `curl -sL -o /s3-echoer https://github.com/mhausenblas/s3-echoer/releases/latest/download/s3-echoer-linux && chmod +x /s3-echoer && echo This is an in-cluster test | /s3-echoer ${bName} && sleep 3600`],
-                        env: [
-                            {name: "AWS_DEFAULT_REGION", value: `${region}`},
-                            {name: "ENABLE_IRP", value: "true"},
-                        ],
-                    },
-                ],
-            },
-        }, { provider: provider },
-    );
-});
+const s3Pod = new k8s.core.v1.Pod(podName,
+    {
+        metadata: {labels: labels, namespace: appsNamespaceName},
+        spec: {
+            serviceAccountName: sa.metadata.name,
+            containers: [
+                {
+                    name: podName,
+                    image: "amazonlinux:2018.03",
+                    command: ["sh", "-c",
+                        pulumi.interpolate`curl -sL -o /s3-echoer https://git.io/JfnGX && chmod +x /s3-echoer && echo This is an in-cluster test | /s3-echoer ${bucketName} && sleep 3600`,
+                    ],
+                    env: [
+                        {name: "AWS_DEFAULT_REGION", value: pulumi.interpolate`${regionName}`},
```

Comment:
I'd expect the following to work:
```suggestion
                        {name: "AWS_DEFAULT_REGION", value: regionName},
```

Reply:
It does not unfortunately. I tried this as-is before using interpolation and the Pod would error. Here's the `kubectl describe on the pod`:

```
      AWS_DEFAULT_REGION:           Calling [toString] on an [Output<T>] is not supported.

                                    To get the value of an Output<T> as an Output<string> consider either:
                                    1: o.apply(v => `prefix${v}suffix`)
                                    2: pulumi.interpolate `prefix${v}suffix`

                                    See https://pulumi.io/help/outputs for more details.
                                    This function may throw in a future version of @pulumi/pulumi.
      ENABLE_IRP:                   true
      AWS_ROLE_ARN:                 arn:aws:iam::153052954103:role/s3-0cd6cf6
      AWS_WEB_IDENTITY_TOKEN_FILE:  /var/run/secrets/eks.amazonaws.com/serviceaccount/token
```

Reply:
Correction, I had used the string value of the `Output<string` using `${regionName}` hence the error. Will correct